packages/amplify-cli/src/snapshot-serializer.ts

```diff
@@ -0,0 +1,12 @@
+const originalPlatform = process.platform;
+// converts inputs to toMatchInlineSnapshot() function into supported format
+export function snapshotSerializer(){
```

Comment:
Let's export the serializer object as `windowsPathSerializer`, and then any tests that need to use it can call `expect.addSnapshotSerializer()`. It also allows us to potentially export more serializers from this file in the future.